app/views/provider/admin/user/access_tokens/_form.html.slim

```diff
@@ -7,3 +7,14 @@
 = form.input :permission, as: :patternfly_select,
                           collection: @access_token.available_permissions,
                           include_blank: false
+
+- if @access_token.persisted?
```

Comment:
This is the kind of stuff that belongs to the presenter IMHO. Also, it's not recommended to use instance variables in partials, pass it as a local instead.

Reply:
The form is loaded from `#new` and `#edit` actions, and they would probably need a different presenter each. I didn't move this logic to a presenter to avoid creating a new presenter for #edit and having to duplicate a new method `access_token_persisted?` in both presenters.

I moved the instance var to a local, though: https://github.com/3scale/porta/pull/3943/commits/4acb540acc7dbe4a110625d6a73d4e8f174a478b